test/test_helper.rb

```diff
@@ -3,10 +3,13 @@
 
 require 'minitest/autorun'
 require 'active_record'
-
+require 'pry'
```

Comment:
I spy a little pry

Reply:
Hahah yeah this one is actually deliberate. I'd like it to be included in the test env moving forward.